generators/app/templates/src/utils/navUtils.js

```diff
@@ -0,0 +1,10 @@
+export const getCurrentRouteName = navigationState => {
```

Comment:
Does this relate to https://github.com/Wolox/wolmo-bootstrap-react-native/issues/42 ?

Reply:
Mmm not really... it used where you say and also here. The intention is that it can be used for several situations and ask when you want the current route.